fem/fe_coll.cpp

```diff
@@ -162,6 +162,11 @@ FiniteElementCollection *FiniteElementCollection::New(const char *name)
    {
       fec = new H1Pos_FECollection(atoi(name + 10), atoi(name + 6));
    }
+   else if (!strncmp(name, "H1Ser_", 6))
+   {
+      // Are the numbers 10 and 6 correct here?
+      fec = new H1Ser_FECollection(atoi(name + 10), atoi(name + 6));
+   }
```

Comment:
The numbers 10 and 6 here refer to the position in the string that is the name of the collection (used e.g. when a gridfunction is written/read from a file).

So if the string is `H1Ser_2D_P3`, see line 1530 below, then `atoi(name + 6)` is 2 and `atoi(name + 10)` is 3.